doc/nrf/ug_thingy53.rst

```diff
@@ -285,3 +273,48 @@ To build and program the source code from the command line, complete the followi
          west flash
 
    The device resets and runs the programmed sample or application.
+
+.. _thingy53_app_guide:
+
+Thingy:53 application guide
+***************************
+
+Thingy:53 does not have a built-in J-Link debug IC.
+Because of that, the samples and applications that are compatible with Thingy:53 by default use MCUboot bootloader with serial recovery support.
+Enabling DFU and MCUboot requires consistency in configuration of the samples and applications.
+
+The samples and applications that comply with Thingy:53 aplication must comply with the following configuration:
+
+1. Partition manager configuration
+   Thingy:53 compatible samples use :ref:`partition manager` to define memory paritions.
+   To ensure that partition layout would not change between builds, sample must use static partition layout that is the same as for Thingy:53 compatible samples in NCS.
+   The FLASH layout consistency is required to allow MCUboot to perform proper image updates and settings storage partition erase.
+
+   The PCD SRAM partition is locked by the MCUboot bootloader to prevent application from modifying network core firmware.
```

Comment:
```suggestion
   The PCD SRAM partition is locked by the MCUboot bootloader to prevent the application from modifying the network core firmware.
```